app/views/completed_transaction/show.html.erb

```diff
@@ -8,34 +8,36 @@
   publication: @publication,
   edition: @edition,
 } do %>
-  <% if @publication.promotion_choice == 'organ_donor' %>
-    <div id="organ-donor-registration-promotion">
-      <p>Please join the NHS Organ Donor Register.</p>
-      <p>If you needed an organ transplant would you have one? If so please help others.</p>
-      <p>If you live in Wales and want to be an organ donor, you don’t need to do anything. Find out about your choices for <a href="https://www.organdonation.nhs.uk/supporting-my-decision/welsh-legislation-what-it-means-for-me/" rel="external">organ donation in Wales</a>.</p>
-      <p>
-        <%= link_to 'Join', @publication.promotion_url,
-              title: "Register to become an organ donor", rel: "external", class: "button", role: "button" %>
-      </p>
-    </div>
-  <% elsif @publication.promotion_choice == 'register_to_vote' %>
-    <div id="register-to-vote-promotion"
-         data-module="auto-track-event"
-         data-track-category="linkTrack"
-         data-track-action="linkDisplayed"
-         data-track-label="Register">
-      <p>You must <a href="/register-to-vote">register to vote</a> by 7 June if you want to take part in the EU referendum. You can register online and it only takes 5 minutes.</p>
-      <p>
-        <%= link_to 'Register', @publication.promotion_url,
-              title: "Register to vote", rel: "internal", class: "button", role: "button",
-              data: {
-                module: 'track-click',
-                track_category: 'linkTrack',
-                track_action: 'linkClicked',
-                track_label: @publication.promotion_url
-              } %>
-      </p>
-    </div>
+  <% if @publication.promotion %>
+    <% if @publication.promotion['category'] == 'organ_donor' %>
```

Comment:
There aren't any methods available in the CompletedTransactionPresenter.  @binaryberry and I did discuss extracting this out in to methods, but felt there wasn't enough duplication going on to warrant it.